packages/ui/components/core/VerifiedBadge.tsx

```diff
@@ -0,0 +1,61 @@
+import { Icon } from '@iconify/react'
+import { Avatar, Tooltip, Text, Group, createStyles, Badge } from '@mantine/core'
+
+// tooltip max characters for width limit
+const MAX_CHARACTERS = 80
+
+const useStyles = createStyles((theme) => ({
+	avatar: {},
+	badge: {
+		paddingLeft: 0,
+		paddingRight: 0,
+		borderStyle: 'hidden',
+	},
+	icon: {
+		color: 'white',
+		position: 'absolute',
+		fontWeight: 500,
+		height: '17.5px',
+		width: '17.5px',
+	},
+	text: {
+		width: 'auto',
+		fontSize: '16px',
+		lineHeight: '20px',
+		fontWeight: 500,
+		color: 'black',
+		marginLeft: '9.25px',
+	},
+	tooltip: {},
+}))
+
+export const VerifiedBadge = ({ text, tooltip_text }: Props) => {
+	const { classes } = useStyles()
+	const trigger_max_width = tooltip_text.length > MAX_CHARACTERS ? 600 : 'auto'
+
+	const verified_avatar = (
+		<Avatar size={24} radius={100} className={classes.avatar} color='green' variant='filled'>
```

Comment:
changed color to themed value (https://mantine.dev/theming/colors/#colors-index-reference)